docs/content/users/extend/customizing-images.md

```diff
@@ -80,6 +80,33 @@ ENV COMPOSER_HOME=""
 
 **Remember that the Dockerfile is building a Docker image that will be used later with DDEV.** At the time the Dockerfile is executing, your code is not mounted and the container is not running, it’s just being built. So for example, an `npm install` in `/var/www/html` will not do anything useful because the code is not there at image building time.
 
+### Available Environment Variables at Build Time
+
+At build time the web Dockerfile can use:
```

Comment:
Slightly more specific—saying this here rather than in the title:

```suggestion
The following environment variables are available for the web Dockerfile to use at build time:
```